ui/src/pages/ShoeMap.tsx

```diff
@@ -1,10 +1,11 @@
 import * as React from 'react';
+import {typography} from "../styles";
 
 export const ShoeMap = ()=> {
     return (
         <div>
-          <h1>ShoeMap</h1> 
-          <span> test text fkjbkjbjdbvdjfv</span> 
+          <typography.TitleText>ShoeMap</typography.TitleText> 
```

Comment:
I'd rather the pattern be import this from a file itself, it helps tree shaking + readability.

Destructuring >> object access IMO.